src/modules/AccountSettings/QuestionCard.tsx

```diff
@@ -93,6 +94,14 @@ export const QuestionPanel = ({
                   <Lock className="w-4 h-4" />
                 )}
               </Toggle>
+            </div>
+
+            <CardDescription className="flex gap-1 items-center">
+              {question.public ? (
+                <Unlock className="w-4 h-4" />
+              ) : (
+                <Lock className="w-4 h-4" />
+              )}
```

Comment:
reverted this because moved the toggle, feels weird without icon.
alternatively we can remove this section entirely, and give tooltip to the button toggler